### DISCUSSION TITLE
i have question of feature requests

### DISCUSSION THREAD

**Discussion by nyaruru1** [PaperMC/Paper#11578] [Ideas]
### Is your feature request related to a problem?

no it only for performance maybe

### Describe the solution you'd like.

to day i found mod name ("Async Mod") In Fabric 
This Mod it say benchmark with 3500 Villager in Intel Core i7-10700

Purpur(Maybe it PaperFork) TPS Is 4.61 & MSPT 150++
but This Mod is TPS 20 & MSPT 40

i want to know it can patch to paper or not It might be more efficient if there is no conflict with the paper. 

Thanky You

### Describe alternatives you've considered.

It might be useful for large SMP servers.

**Comment by electronicboy** [comment]
This is not tenable without breaking a large swath of plugins

**Reply by nyaruru1** [reply]
It Ok Thanks
